src/app/js/fields/FieldGrid.js

```diff
@@ -88,6 +93,9 @@ const useStyles = makeStyles({
     },
 });
 
+// 1rem = 16px
+const ROOT_PADDING = 16;
```

Comment:
Maybe we could take advantage of this to use the same variable in the styles.
Also, 1rem != 16px, it's relative to the font-size, font-size change = padding change